internal/security/fileprovider/init.go

```diff
@@ -65,6 +65,7 @@ func BootstrapHandler(_ context.Context, _ *sync.WaitGroup, _ startup.Timer, dic
 
 	if err != nil {
 		lc.Error(fmt.Sprintf("error occurred generating tokens: %s", err.Error()))
+		os.Exit(1)
```

Comment:
Note: may be able to remove this at some point... current bootstrap does not support returning non-zero exit status.

Reply:
good note..maybe we should put that comment into the code.

Reply:
Michael Estrin provided me a comment offline that the cleaner way to do this is to make BoostrapHandler a method of an object with an ExitCode() method, then in main() retrieve the exit code and exit with it.  I will go that route.

Reply:
Rewritten to set exitCode member variable.